routes/products.js

```diff
@@ -23,4 +41,55 @@ router.get('/:id', (req, res) => {
 		});
 });
 
+
+router.get('/', (req, res) => {
```

Comment:
This seems strange - why are there two gets here? We should look at breaking the second down a bit too. There's a lot of nested promises here that will be a nightmare to handle. Overall, you're going in a good direction though.